### DISCUSSION TITLE
Using .browserslistrc file over package.json

### DISCUSSION THREAD

**Discussion by yordis** [vercel/next.js#17316] [Help] [answered]
Hey there,

I just read the documentation of Next about Browserslist configuration, and also search in the entire repo for information about it.

I don't see anywhere that I could use `.browserslistrc` over `package.browserslist` configuration.

Is this possible? 

It would be good to document about the usage of the file itself instead of using `package.browserslist`.

**Comment by jaydenseric** [comment, marked as answer]
See also https://github.com/vercel/next.js/discussions/12826 .
